Pages/socialButtonsPage.js

```diff
@@ -0,0 +1,23 @@
+import { Selector, t } from "testcafe";
```

Comment:
Social buttons don't need their separate page. All of this could be included to the product page. Selectors and methods are good.

Reply:
This comment is still relevant.